src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -29,23 +28,23 @@ const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc" }: Props) =>
     }, new Array<string>())
     .join("&")
 
-  const classes = useCustomStyles()
-
   return courtCases.length === 0 ? (
     <Paragraph>{"There are no court cases to show"}</Paragraph>
   ) : (
-    <Table head={<CourtCaseListTableHeader order={order} />}>
-      {courtCases.map((courtCase, index) => (
+    <table className="cases-list">
```

Comment:
We use `Table` from `govuk-react` in quite a few places, e.g. the rows are all `Table.Row`s. It would be nice to have consistency across them, and the govuk table handles layout nicely for us. Is it possible to use the govuk-react `Table` here or do we need to use the plain HTML table?

Reply:
we are using 'table' because we don't want to use the styling applied from 'Table'